test/quantization/test_quantized_op.py

```diff
@@ -2702,6 +2702,100 @@ def test_qrnncell(self, num_batches, input_size, hidden_size, per_channel_quant)
                 self.assertEqual(result_ref[0], result_dynamic[0], msg="torch.quantized_rnncell results are off")
 
 
+"""Tests recurrent layers."""
+class TestPTQRNN(TestCase):
+    def _snr(self, x, x_hat):
+        if isinstance(x, (list, tuple)):
+            assert(len(x) == len(x_hat))
+            res = []
+            for idx in range(len(x)):
+                res.append(self._snr(x[idx], x_hat[idx]))
+            return res
+        if x_hat.is_quantized:
+            x_hat = x_hat.dequantize()
+        noise = (x - x_hat).square().mean().item()
+        if noise == 0:
+            return 0.0, float('inf'), float('inf')
+        signal = x.square().mean().item()
+        snr = signal / noise
+        snr_db = 10 * np.log10(snr)
+        return signal, noise, snr_db
+
+    @override_qengines
+    def test_lstm(self):
```

Comment:
In this case, we are testing only for (1) does the module get created correctly and (2) does it work with the quantization workflow. 
For (1), please add a test in test_quantized_module, this should check module creation, serialization, attributes etc.
For (2), please add a test in test_quantize, which should call prepare and convert and check that module swap occured as expected. 
We do not need to check numerics as we are not creating a new op here, suggest removing this test from this file.

Reply:
Numerics have to be checked, as the functionality of the LSTM is not just dependent on the linear, but also on the "wiring". If there is no unittest for numerical equivalence, it might perform a different function